GPT/gpt.py

```diff
@@ -113,6 +113,10 @@ def add_to_confirmation_gui(model_output: str):
         GPTState.text_to_confirm = model_output
         confirmation_gui.show()
 
+    def additional_user_context():
```

Comment:
Since we are already adding an additional context variable, let's change this to contextual user context or something similar perhaps?